### DISCUSSION TITLE
Create a plugin

### DISCUSSION THREAD

**Discussion by omerkimel** [vue-leaflet/vue-leaflet#205] [Q&A]
Hi i would like to know if there is any spacific way I shoud approce wrapping a leaflet plugin for vue 3?

**Comment by mikeu** [comment]
Hi @omerkimel , thank you for this important question, and sorry for my late reply. We do not have any standard approach or documentation on this yet, but are aware that it is a key piece of functionality. The next stage of development will address some of the difficulties around [mixing Leaflet with SSR](https://github.com/vue-leaflet/vue-leaflet/discussions/161). That will provide better support for the standard (i.e. non-SSR) version by default, and hopefully make plugin development (and porting from Vue2Leaflet) somewhat simpler.

**Reply by xnhp** [reply]
Hey, have there been any updates on this already?
